proposals/records.md

```diff
@@ -48,9 +48,9 @@ The synthesized members are as follows:
 
 If the record is derived from `object`, the record type includes a synthesized readonly property equivalent to a property declared as follows:
 ```C#
-protected Type EqualityContract { get; };
+Type EqualityContract { get; };
 ```
-The property is `virtual` unless the record type is `sealed`.
+The property is `private` if the record type is `sealed`. Otherwise, the property is `virtual` and `protected`.
```

Comment:
It is silly to have an `EqualityContract` property for a sealed record type that derives from `object`. It serves no purpose.

Reply:
> It is silly to have an EqualityContract property for a sealed record type that derives from object. It serves no purpose.

There is a sentence below: "Can we omit EqualityContract if the record type is sealed and derives from System.Object?"